src/ui/server/controllers/insurance/declarations/confidentiality/save-and-back/index.test.ts

```diff
@@ -0,0 +1,105 @@
+import { post } from '.';
+import { ROUTES } from '../../../../../constants';
+import { Request, Response } from '../../../../../../types';
+import save from '../save-data';
+import { mockApplication, mockReq, mockRes } from '../../../../../test-mocks';
+
+const {
+  INSURANCE: { INSURANCE_ROOT, ALL_SECTIONS },
+} = ROUTES;
+
+describe('controllers/insurance/declarations/confidentiality/save-and-back', () => {
+  let req: Request;
+  let res: Response;
+
+  jest.mock('../save-data');
+
+  let mockSaveData = jest.fn(() => Promise.resolve(true));
+  save.declaration = mockSaveData;
+
+  const refNumber = Number(mockApplication.referenceNumber);
+
+  const mockFormBody = {
+    _csrf: '1234',
+    mock: true,
+  };
+
+  beforeEach(() => {
+    req = mockReq();
+    res = mockRes();
+
+    res.locals.application = mockApplication;
+    req.params.referenceNumber = String(mockApplication.referenceNumber);
+
+    req.body = mockFormBody;
+  });
+
+  describe('when the form has data', () => {
+    it('should call mapAndSave.policyAndExport with application and req.body', async () => {
```

Comment:
typo

Reply:
As discussed will fix this in the next PR